StRoot/StETofMatchMaker/StETofMatchMaker.h

```diff
@@ -220,9 +229,17 @@ class StETofMatchMaker : public StMaker {
     std::map< Int_t, Int_t >  mClockJumpDirection;
 
     std::string                    mHistFileName;
+      
     std::map< std::string, TH1* >  mHistograms;
     std::map< std::string, TH2* >  mHistograms2d;
 
+    // used for single sided match cases
+    Double_t   dx_3sig; 
```

Comment:
Member variables should follow the STAR naming conventions.  Something like mDX3Sig... (start w/ an "m", avoid "_", CamelCase).